packages/shared/type.ts

```diff
@@ -13,6 +13,9 @@ export interface Config {
         sort: Sort;
         query: string;
     }
+    filename: string;
+    fileTemplateText: string;
```

Comment:
Das klingt so, als könnte man hier auch template variablen benutzen, ist aber doch nur nen string.
```suggestion
    defaultFileContent: string;
```